Test.C

```diff
@@ -128,20 +128,22 @@ int main(){
     player.currentCredits = 30;
 
     while(player has options and more rounds exists){
-
-        //output player status (if statements)
-        //current credit
-        //current equipment
-        //pulls from info from index amount table referencing index struct table
-                /* WHY DO WE NEED TO USE MULTIPLE ARRAYS FOR EQUIPMENT INSTEAD OF A LIST WITH A QUANTITY ATTRIBUTE? */
-        //current encrypt
-        //iterate and print through current encryption
+        //receive credits NATHAN
+        if(end of round){ //output player status (if statements)
```

Comment:
This if statement is unnecessary since the start of the loop iteration indicates the end of the previous round. This if statement would make sense if we were implementing a GUI since then a completion of the main loop would just be a frame for the GUI, and not necessarily an entire round. 
```suggestion
```